crates/turborepo-filewatch/src/globwatcher.rs

```diff
@@ -24,26 +25,53 @@ pub struct GlobSet {
     exclude: Any<'static>,
 }
 
+#[derive(Debug, Error)]
+pub struct GlobError {
+    // Boxed to minimize error size
+    underlying: Box<wax::BuildError>,
+    raw: String,
```

Comment:
Maybe `glob` or `input` would be slightly more descriptive than `raw`?

Reply:
changed to `raw_glob`